scripts/assets/accessibility_label_exemptions.textproto

```diff
@@ -8,6 +8,7 @@ exempted_activity: "app/src/main/java/org/oppia/android/app/home/HomeActivity"
 exempted_activity: "app/src/main/java/org/oppia/android/app/home/recentlyplayed/RecentlyPlayedActivity"
 exempted_activity: "app/src/main/java/org/oppia/android/app/mydownloads/MyDownloadsActivity"
 exempted_activity: "app/src/main/java/org/oppia/android/app/player/state/testing/StateFragmentTestActivity"
+exempted_activity: "app/src/main/java/org/oppia/android/app/policies/PoliciesActivity"
```

Comment:
This shouldn't be exempted; we should be defining an accessibility label for this page (presumably a default with an override at runtime).

Reply:
Done.